src/App.tsx

```diff
@@ -1,23 +1,70 @@
 import styles from "./App.module.css";
+import FlightTable from "./components/FlightTable/FlightTable";
+import { faker } from "@faker-js/faker";
+import React from "react";
+import Button from "@mui/material/Button";
 
 export interface Flight {
   // ... fields here
+  flightNumber: string;
+  origin: string;
+  destination: string;
+  departureTime: string;
+  arrivalTime: string;
+  flightDuration: number;
 }
 
 const App = () => {
   // state for flights should go here
+  const [flightList, setFlightList] = React.useState<Flight[]>([]);
+
+  function formatAMPM(date: Date): string {
```

Comment:
Clever function! Let's use "arrow notation" to make our functions! With ES6 Javascript (the newest version), the standard is to do this instead of the `function` word:

```diff
- function formatAMPM(date: Date): string {
+ const formatAMPM = (date: Date): string => {
```